odoo/addons/base/models/assetsbundle.py

```diff
@@ -415,6 +452,138 @@ def js_with_sourcemap(self):
+
+        :param show_inherit_info: if true add the file url and inherit
+            information in the template.
+        :return ir.attachment representing the content of the bundle XML
+        """
+        template_dict = OrderedDict()
+        parser = etree.XMLParser(ns_clean=True, recover=True, remove_comments=True)
+
+        for asset in self.templates:
+            # Load content.
+            try:
+                content = asset.content.strip()
+                template = content if content.startswith('<odoo>') else f'<templates>{asset.content}</templates>'
+                io_content = io.BytesIO(template.encode('utf-8'))
+                content_templates_tree = etree.parse(io_content, parser=parser).getroot()
+            except etree.ParseError as e:
+                _logger.error("Could not parse file %s: %s", asset.url, e.msg)
+                raise
+            addon = asset.url.split('/')[1]
+            template_dict.setdefault(addon, OrderedDict())
+            # Process every templates.
+            for template_tree in list(content_templates_tree):
+                template_name = None
+                if 't-name' in template_tree.attrib:
+                    template_name = template_tree.attrib['t-name']
+                    dotted_names = template_name.split('.', 1)
+                    if len(dotted_names) > 1 and dotted_names[0] == addon:
+                        template_name = dotted_names[1]
+
+                if 't-inherit' in template_tree.attrib:
+                    inherit_mode = template_tree.attrib.get('t-inherit-mode', 'primary')
+                    if inherit_mode not in ['primary', 'extension']:
+                        raise ValueError(_("Invalid inherit mode. Module %r and template name %r") % (addon, template_name))
+
+                    # Get inherited template,  # Get inherited template, the identifier can be "addon.name", just "name" or (silly) "just.name.with.dots"
+                    parent_dotted_name = template_tree.attrib['t-inherit']
+                    split_name_attempt = parent_dotted_name.split('.', 1)
+                    parent_addon, parent_name = split_name_attempt if len(split_name_attempt) == 2 else (addon, parent_dotted_name)
+                    if parent_addon not in template_dict:
+                        if parent_dotted_name in template_dict[addon]:
+                            parent_addon = addon
+                            parent_name = parent_dotted_name
+                        else:
+                            raise ValueError(_("Module %r not loaded or inexistent (try to inherit %r), or templates of addon being loaded %r are misordered (template %r)") % (parent_addon, parent_name, addon, template_name))
+                    if parent_name not in template_dict[parent_addon]:
+                        raise ValueError(_("Cannot create %r because the template to inherit %r is not found.") % (f'{addon}.{template_name}', f'{parent_addon}.{parent_name}'))
```

Comment:
The new `_` function takes injectable parameters as argument
```suggestion
                        raise ValueError(_("Cannot create %r because the template to inherit %r is not found.", f'{addon}.{template_name}', f'{parent_addon}.{parent_name}'))
```